### DISCUSSION TITLE
Artificially Split Display

### DISCUSSION THREAD

**Discussion by AaronWNW** [mrkai77/Loop#507] [Q&A]
Hey,
don't know if I just missed the feature in the settings but here's my problem/idea:
I want to buy an ultra wide monitor and like to keep on using Loop. The Problem is that I want to artificially split the monitor in two parts for Loop as if I hade two separate displays. The problem would be that the monitor would be too wide to make small windows, and I could only have two windows next to each other instead of maybe 4.

Hope you understood what im trying to say :)
I'd like to hear your thoughts on this.

Best regards, 
Aaron W.

**Comment by mrkai77** [comment]
Would custom window actions not fit your needs?
<img width="302" alt="Screenshot 2024-07-14 at 3 25 06 PM" src="https://github.com/user-attachments/assets/fee7837d-5b70-4cce-95f2-5410574cf8b5">

**Reply by AaronWNW** [reply]
Sorry I can't find the setting. Is it in Loop or from the OS?

**Reply by mrkai77** [reply]
This is in Loop, in the keybinds tab when hovering over a keybind item.

**Reply by AaronWNW** [reply]
I didn't see these settings sorry. 
A option to have horizontal quarters instead would be my use case. I don't know if im the only one with this problem so I don't know if its really actually worth implementing :)

Thx for the fast response

**Reply by AaronWNW** [reply]
But the right Cycle more or less does the job but it would be thirds not quarters

**Reply by mrkai77** [reply]
<img width="1251" alt="Screenshot 2024-07-15 at 2 19 20 AM" src="https://github.com/user-attachments/assets/ed4e21f6-f147-4f37-a486-e36f78e22be6">

Maybe this will give you a better look at where the setting is located? Using custom keybinds, you will be able to make horizontal quarters as so: 

<img width="474" alt="Screenshot 2024-07-15 at 2 20 25 AM" src="https://github.com/user-attachments/assets/e5c37a48-26f0-4fa8-9c4b-9dbe208d03b2">

**Reply by AaronWNW** [reply]
Okay cool :), thx for the answer
